scanomatic/io/rpc_client.py

```diff
@@ -83,9 +83,8 @@ def launch_local(self) -> None:
             Popen(["scan-o-matic_server"])
         else:
             self._logger.warning(
-                "Can't launch because server is {0}".format(
-                    ['not local', 'online'][self.online],
-                )
+                "Can't launch because server is %s",
```

Comment:
don't introduce `%` formatting rather use `f"..."`

Reply:
It's not so much a preference of type of string formatting interpolation to use, it's a matter of using the features that logging offers. You don't want string interpolation to happen if the message isn't going to be published.

Reply:
I understand the argument but I don't agree, I think the readability of `%` formatting use so much worse than f-strings, which is a general tendency not only in python to format that way that I think there's a strong argument to do string formatting even if it is useless when not output. That sound like chasing performance that shouldn't be that important

Reply:
I don't find it very hard to read the %-formatting, not using a module as intended disturbs me more.

Reply:
I'm sorry but I don't agree it is much harder to follow for me, and if we should do such a change we should do it after agreeing on it and reworking it across all files.

Reply:
OK, I'll back out changes in logging.